packages/web/src/validators/input-validators/input-validators.ts

```diff
@@ -1,15 +1,31 @@
 import { Validator } from "../validator";
 
+let emailPattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
 export const requiredField: Validator<string> = {
-    validate: (value: string) => {
-        return value != null && value.trim() != "" ? true : false;
-    },
-    errorMessage: { "en": "Please complete the required field to continue.",  "fr": "Veuillez compléter les champs obligatoires afin de continuer." }
+  validate: (value: string) => {
+    return value != null && value.trim() != "" ? true : false;
+  },
+  errorMessage: { "en": "Enter information to continue.",  "fr": "Saisissez des renseignements pour continuer." }
+}
+
+export const requiredEmailField: Validator<string> = {
+  validate: (value: string) => {
+    return (value != null && value.trim() != "" && value.toLowerCase().match(emailPattern)) ? true : false;
+  },
+  errorMessage: { "en": "Enter a valid email address to continue. Use a standard format. Example: name@address.ca.",  "fr": "Saisissez votre adresse courriel pour continuer. Utilisez un format standard. Exemple: nom@adresse.ca." }
```

Comment:
Can you move the `en` and the `fr` each to its own line to make it easier to read?